src/mpid/ch4/ch4_api.txt

```diff
@@ -26,8 +26,8 @@ Non Native API:
       NM : comm_ptr
      SHM : comm_ptr
   mpi_open_port : int
-      NM : info_ptr, port_name-2
-     SHM : info_ptr, port_name-2
+      NM : info, port_name-2
+     SHM : info, port_name-2
   mpi_close_port : int
```

Comment:
Change of input pointer parameter is not described in the commit message. Maybe better to separate to a different commit.

Reply:
Will do

Reply:
Done.